src/sidebar/store/modules/annotations.js

```diff
@@ -545,6 +543,35 @@ function savedAnnotations(state) {
   });
 }
 
+/**
+ * @typedef AnnotationsStore
+ *
+ * // Actions
+ * @prop {typeof addAnnotations} addAnnotations
+ * @prop {typeof clearAnnotations} clearAnnotations
+ * @prop {typeof focusAnnotations} focusAnnotations
+ * @prop {typeof hideAnnotation} hideAnnotation
+ * @prop {typeof removeAnnotations} removeAnnotations
+ * @prop {typeof unhideAnnotation} unhideAnnotation
+ * @prop {typeof updateAnchorStatus} updateAnchorStatus
+ * @prop {typeof updateFlagStatus} updateFlagStatus
+ 
+ *
+ * // Selectors
+ * @prop {() => number} annotationCount
+ * @prop {(id: string) => boolean} annotationExists
+ * @prop {(id: string) => Annotation} findAnnotationByID
+ * @prop {(tags: string[]) => string[]} findIDsForTags
+ * @prop {() => string[]} focusedAnnotations
+ * @prop {() => string[]} highlightedAnnotations
```

Comment:
The fact that `newAnnotations` returns `Annotation`s whereas `focusedAnnotations` returns IDs is surprising, but this PR is just surfacing an existing issue.

Reply:
indeed, I'll do a follow up refactor on that